Jenkinsfile

```diff
@@ -141,6 +141,26 @@ Map taskGenPyPRT() {
 def taskPrepare(cfg) {
  	cepl.cleanCurrentDir()
 	papl.checkout(REPO, env.BRANCH_NAME, CREDS)
+
+    def deps = readProperties(file: "${SOURCE}/src/dependencies.properties")
+
+	dir(path: "${SOURCE}/src") {
+        Map PYBIND11_CONFIG = [ r: 'thirdparty', g: 'com.github', a: 'pybind11', v: deps.PYBIND11_VERSION, e: 'zip', f: 'pybind11.zip', extract: true ]
+		psl.fetchFromNexus2(PYBIND11_CONFIG)
```

Comment:
We don't need to keep the .zip file. Either delete it after extract or exclude it from the stash call on line 164.

Reply:
done